selftests/deployment/roles/avocado/tasks/rpm/plugins.yml

```diff
@@ -10,10 +10,20 @@
       - python3-avocado-plugins-varianter-yaml-to-mux
     state: latest
 
+- name: Install the Avocado VT plugin using RPM
+  dnf:
+    name: python3-avocado-vt
```

Comment:
ACK.  I wonder why the naming was not followed.  Best bet is Fedora reviewers asked nicely, and package owner followed suit.